test/bind_cdecl_mf_test.cpp

```diff
@@ -1,6 +1,6 @@
 #include <boost/config.hpp>
 
-#ifndef BOOST_MSVC
+#if defined(_WIN32) && (defined(_WIN64) || defined(__MINGW64__))
```

Comment:
It could be just `defined(_WIN64) || defined(__MINGW64__)` but I am not exactly sure.